### ISSUE TITLE
Missing Accessibility

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#46]
Hello! I noticed that the studygroupmap project is missing some accessibility features. It would be great if we could add some accessibility features to make the project more inclusive. Thank you!
